observability/logging/logging-6.0/log6x-upgrading-to-6.adoc

```diff
@@ -0,0 +1,387 @@
+:_mod-docs-content-type: ASSEMBLY
+include::_attributes/common-attributes.adoc[]
+[id="log6x-upgrading-to-6"]
+= Upgrading to Logging 6.0
+:context: log6x
+
+toc::[]
+
+Logging v6.0 is a significant upgrade from previous releases, achieving several longstanding goals of Cluster Logging:
+
+* Introduction of distinct operators to manage logging components (e.g., collectors, storage, visualization).
+* Removal of support for managed log storage and visualization based on Elastic products (i.e., Elasticsearch, Kibana).
+* Deprecation of the Fluentd log collector implementation.
+* Removal of support for `ClusterLogging.logging.openshift.io` and `ClusterLogForwarder.logging.openshift.io` resources.
+
+[NOTE]
+====
+The *cluster-logging-operator* does not provide an automated upgrade process.
+====
+
+Given the various configurations for log collection, forwarding, and storage, no automated upgrade is provided by the *cluster-logging-operator*. This documentation assists administrators in converting existing `ClusterLogging.logging.openshift.io` and `ClusterLogForwarder.logging.openshift.io` specifications to the new API. Examples of migrated `ClusterLogForwarder.observability.openshift.io` resources for common use cases are included.
+
+
+== Log Storage
+
+The only managed log storage solution available in this release is a Lokistack, managed by the *loki-operator*. This solution, previously available as the preferred alternative to the managed Elasticsearch offering, remains unchanged in its deployment process.
+// Something recently changed in the code re the Lokistack CR relating to the CLF?
+
+[IMPORTANT]
+====
+To continue using an existing Red Hat managed Elasticsearch deployment provided by the *elasticsearch-operator*, remove the owner references from the `Elasticsearch` resource named `elasticsearch` in the `openshift-logging` namespace before removing the `ClusterLogging` resource named `instance` in the same namespace.
+====
+
+== Log Visualization
+
+The OpenShift console UI plugin for log visualization has been moved to the *cluster-observability-operator* from the *cluster-logging-operator*.
+// Pending support statement.
+
+[IMPORTANT]
+====
+To continue using an existing Red Hat managed Kibana deployment provided by the *elasticsearch-operator*, remove the owner references from the `Kibana` resource named `kibana` in the `openshift-logging` namespace before removing the `ClusterLogging` resource named `instance` in the same namespace.
```

Comment:
The steps for removing the references are in the article https://access.redhat.com/articles/6991632, the section "Disconnect Elasticsearch and Kibana CRs from ClusterLogging".

This is an important step and it deserves to be documented.